ui/src/screens/Post/components/NewPostForm.tsx

```diff
@@ -30,11 +31,15 @@ const Buttons = styled.div`
 `;
 
 const INITIAL_TYPE = "talent";
+const EDIT_POST_PATHNAME = "/posts/edit";
+const NEW_POST_PATHNAME = "/posts/new";
```

Comment:
Instead of this messiness with checking path strings, I think it's better to just pass a prop at the router level.

In `App.tsx`:
```
<ProtectedRoute path="/posts/edit" component={PostScreen} edit={true} />
```

This is abit more complicated because there's a custom `ProtectedRoute` component. I can work on this as a lower priority.

Reply:
Sure, let me know if you need help